### PR TITLE
Update DHIS2 App with Partial Country Drop Down

### LINKED ISSUE #58: Update DHIS2 App with Partial Country Drop Down

### PR DESCRIPTION
**Pull request by EmmanuelKoh**
Closes #58

### CODE REVIEW

File: src/api/crosscutRequests.js [hunk 1]
```diff
@@ -281,15 +281,89 @@ export const fetchSupportedBoundaries = async () => {
...
+    // return (
+    //   resp.boundaryList
+    //     .filter((bound) => bound.featureFlags.includes('all'))
+    //     // .filter((bound) => bound.entireCountry === true)
+    //     .sort((a, b) => {
+    //       if (a.countryName > b.countryName) return 1;
+    //       if (a.countryName < b.countryName) return -1;
+    //       return 0;
+    //     })
+    // );
```
Comment (briannapoulos) [inline src/api/crosscutRequests.js:366]: Please remove the commented out code

File: src/api/crosscutRequests.js [hunk 2]
```diff
@@ -281,15 +281,89 @@ export const fetchSupportedBoundaries = async () => {
...
+    // sort/organize the boundary list
+    const boundaryList = Object.values(resp.boundaryList);
 
-    return resp.boundaryList
-      .filter((bound) => bound.featureFlags.includes('all'))
-      .filter((bound) => bound.entireCountry === true)
-      .sort((a, b) => {
-        if (a.countryName > b.countryName) return 1;
-        if (a.countryName < b.countryName) return -1;
+    const entireCountries = boundaryList.filter(
```
Comment (briannapoulos) [inline src/api/crosscutRequests.js:287]: Add a filter for entire country and partial country to only include feature flags that include `all`

Comment (EmmanuelKoh) [reply src/api/crosscutRequests.js:287]: [f61a280](https://github.com/crosscutio/dhis2-crosscut-app/pull/59/commits/f61a2803ebba6c7b375140a3b98b216405c794a3)

**Review by briannapoulos — APPROVED** [review]
This worked well for me.

**Pull request merged**